### ISSUE TITLE
Invalid option error in install-driver when using driver-options

### ISSUE THREAD

**Issue by nathansalaun-sofia** [Lightricks/Kintsugi#99]
I can't seem to make `install-driver` work with `driver-options`, the command `kintsugi install-driver --interactive-resolution false` fails with the following error:

```
~ % kintsugi install-driver --interactive-resolution false
/Library/Ruby/Gems/2.6.0/gems/kintsugi-0.7.0/lib/kintsugi.rb:40:in `parse_options!': invalid option: --interactive-resolution (OptionParser::InvalidOption)
        from /Library/Ruby/Gems/2.6.0/gems/kintsugi-0.7.0/lib/kintsugi.rb:23:in `run'
        from /Library/Ruby/Gems/2.6.0/gems/kintsugi-0.7.0/bin/kintsugi:10:in `<top (required)>'
        from /usr/local/bin/kintsugi:23:in `load'
        from /usr/local/bin/kintsugi:23:in `<main>'
~ % 
```

Am I using it wrong?

**Comment by byohay** [comment]
@nathansalaun-sofia sorry, I believe it's a bug! I fixed it in #100

**Comment by nathansalaun-sofia** [comment]
It works now, thank you!
